027-FHIRPoweredHealthcare/Coach/Solution02.md

```diff
@@ -1,46 +1,41 @@
-# Coach's Guide: Challenge 2 - Ingest and stream medical events for post-processing
+# Coach's Guide: Challenge 2 - Extract, transform and load HL7 medical data
 
 [< Previous Challenge](./Solution01.md) - **[Home](./readme.md)** - [Next Challenge>](./Solution03.md)
 
 ## Notes & Guidance
 
-In this challenge, you will deploy a Health Architecture **[FHIR Event Processor](https://github.com/microsoft/health-architectures/tree/master/FHIR/FHIREventProcessor)** serverless solution that provides the following services within **[FHIR Converter](https://github.com/microsoft/FHIR-Converter)**:
+In this challenge, you will deploy a Health Architecture **[HL7toFHIR Conversion](https://github.com/microsoft/health-architectures/tree/master/HL7Conversion#hl7tofhir-conversion)** serverless solution that provides the following services within **[FHIR Converter](https://github.com/microsoft/FHIR-Converter)** and FHIR Proxy:
 - Import and process valid HL7 bundles and persist them to a FHIR Compliant store
 - FHIR Server Proxy connection to FHIR Server without exposing credentials
-- Publish successful FHIR CUD events referencing FHIR Resources to an event hub
+- Publish successful FHIR CUD events referencing FHIR Resources to an event hub to support pre-processing and/or post-processing for event driven workflow and orchestration scenarios.  This can be done by enabling the PublishFHIREventPostProcess module of the FHIR Proxy.
 
 Note: This function is deployed and configured as a part of the HL72FHIR Workflow Platform.
```

Comment:
I would bold and uppercase **NOTE** on this line and all lines in the coach's guide. Brings out the emphasis.